demo/src/app/components/modal/demos/component/modal-component.module.ts

```diff
@@ -8,7 +8,7 @@ import { NgbdModalComponent, NgbdModalContent } from './modal-component';
   imports: [BrowserModule, NgbModule],
   declarations: [NgbdModalComponent, NgbdModalContent],
   exports: [NgbdModalComponent],
-  bootstrap: [NgbdModalComponent],
-  entryComponents: [NgbdModalContent]
+  bootstrap: [NgbdModalComponent]
+  // entryComponents: [NgbdModalContent] // this line would be needed in Angular 8 or older
```

Comment:
It is more about ivy enabled or not - people can be on v9 and have ivy disabled. This situation will continue for a while (until we remove VE)

Reply:
Right. I didn't want to be too technical, and simply assume the defaults here. Not sure if all people know what Ivy is and what they're using without knowing it. But I can of course be more precise. How about this?

> this line is not needed if Ivy is used (i.e., by default, in Angular 9 or newer)

And the the text in the demo:

> In this case, if you're still using ViewEngine (which is the default in Angular 8 or before) remember to add the content component
in the <code>entryComponents</code> section of your <code>NgModule</code>. For Ivy (which is the default in Angular 9 or newer), it's not needed anymore.

Reply:
What about detailing the text in demo without even referring to ViewEngine or Ivy ?

We could state `up to Angular 8` and `Angular 9+` ? at least, no confusing, no technical detail...